app/assets/javascripts/select-kit/addon/components/bulk-select-topics-dropdown.js

```diff
@@ -63,6 +152,97 @@ export default DropdownSelectBoxComponent.extend({
           },
         });
         break;
+      case "archive-topics":
+        this.modal.show(BulkTopicActions, {
```

Comment:
Honestly since these are all the same, IMO we should make a `showBulkTopicActionsModal` function, that accepts a `title`, `action`, and `initialAction` (default null) argument, that way we don't have to repeat all this.